ledger/acctupdates.go

```diff
@@ -1164,13 +1137,10 @@ func (au *accountUpdates) initializeFromDisk(l ledgerForTracker) (lastBalancesRo
 		return
 	}
 
-	// the VacuumDatabase would be a no-op if au.vacuumOnStartup is cleared.
-	au.vacuumDatabase(context.Background())
+	au.accountsq, err = accountsDbInit(au.dbs.Rdb.Handle, au.dbs.Wdb.Handle)
```

Comment:
we really could use a better name for this method. It doesn't initializing anything with the db rather then preparing precompiled sql statements.

Reply:
done